src/main/java/controller/AllUsersServlet.java

```diff
@@ -0,0 +1,35 @@
+package controller;
+
+import Factory.UserServiceFactory;
+import model.User;
+import service.UserService;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.List;
+
+@WebServlet(value = "/users")
+public class AllUsersServlet extends HttpServlet {
+
+  private static UserService userService = UserServiceFactory.getInstance();
+
+  @Override
+  protected void doGet(HttpServletRequest req, HttpServletResponse resp)
```

Comment:
what does this method do?
it need only return list of users

Reply:
After we push SignIn button on start page we need to open users list. Also we need to check if that kind of user exist in our db.
Do I need separate Post method for this? But in that way I would repeat the code.
@Sofasmile